docs/guide/personalization/personalization_quickstart.md

```diff
@@ -1,45 +1,32 @@
-# Quickstart guide
+# Basic service integration
 
-This quickstart guide gives a basic overview of how the personalization engine works and how it is typically enabled.
-Following is an example for the integration in an e-commerce shop.
-
-## How it works
-
-The idea of the recommendation engine is quite simple and built upon four main steps.
-
-A User visits an online shop, navigates through it and leaves some footprints.
-The most popular footprints are selecting and purchasing  products **(1)**.
-This information is sent to the event tracker service of the recommendation engine every time these events happen **(2)**.
-The recommendation engine accumulates the tracked events, categorizes them and calculates recommendations for every product and every user in the shop **(3)**.
-All recommendations are available over the recommender service.
-They can be fetched and presented to users during their sessions.
-
-![Overview of how recommendation works](img/recommendation_overview.png)
-
-#### Integration
-
-Integration is based on the following:
-
-- activation of user tracking (we respect privacy rules and never store any data related to a person, every event is anonymized)
-- embedding recommendations into a website.
+Assuming that you [enabled the Personalization service](enabling_personalization.md), 
+for the service to return recommendations, you must integrate it with your content 
+publishing or eCommerce solution by activating event tracking and embedding 
+recommendation results into the website.
+With the integration configured as described below, you get basic yet effective recommendations.
 
 !!! note
 
-    Below we use the mandator ID '00000' for creating requests.
-    In real-life scenario, you use your own ID that you receive from Ibexa.
+    Examples below use '00000' as a customer ID for creating requests.
+    In a real-life scenario, use the customer ID that you receive from Ibexa.
 
-### Tracking events
+## Tracking events
 
-Step one is called tracking. Every page must call special tracking URL.
-The simplest way is to place a tiny image on every product page, just as it is usually done for analytic tools or visitor counter.
-It looks like this:
+The service primarily relies on event tracking. 
+For the events to be registered, every content item or product page must call 
+a special tracking URL.
+The simplest way to the tracking URL is placing a one pixel image on every page, 
+just like in the case of analytical tools or visitor counters.
+A code that includes an image may look like this:
 
-**`<img href="https://event.yoochoose.net/ebl/00000/click/john.smith/1/123" width="1" height="1">`**
+`<img href="https://event.yoochoose.net/ebl/00000/click/<user_ID>/1/<content_ID>" width="1" height="1">`
```

Comment:
@dabrt if we're changing 123 to <content_ID>, shouldn't we also change content type value? 
so it should be: 
<img href="https://event.yoochoose.net/ebl/00000/click/<user_ID>/<content_type_ID>/<content_ID>" width="1" height="1">`